corehq/form_processor/migrations/0092_auto_20200924_1753.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.13 on 2020-09-24 17:53
+
+from django.db import migrations, models
+
+OLD_NAME = "fp_casetrans_formid_05e599_idx"
```

Comment:
@mkangia I'm curious where this name came from originally? The other index you added to the `ledgertransaction` table was a different format (same as the new name here): https://github.com/dimagi/commcare-hq/blob/30eef3f76e964b888c5f45cab600980b01e90df6/corehq/form_processor/migrations/0091_auto_20190603_2023.py#L7

Reply:
As far as I can make out from the commits (because I have no memory of this really), that name was suggested by django https://github.com/dimagi/commcare-hq/pull/24479/commits/2d12fe4ec66152b37310de9e5e284170e2cc51d9.
I think there was an error when i ran the migration locally due to the length of the index name, https://github.com/dimagi/commcare-hq/pull/24479/commits/4d42a76891cf611099111ee0da23155c222127a6#r290024876.